ha/const.py

```diff
@@ -53,9 +53,15 @@
 CLI_SCHEMA_FILE = "{}/cli_schema.json".format(CONFIG_DIR)
 COMPONENTS_CONFIG_DIR = "{}/components".format(CONFIG_DIR)
 
+# IEM DESCRIPTION string: To be removed
+IEM_DESCRIPTION="WS0080010001,Node, The cluster has lost $host server. System is running in degraded mode. For more information refer the Troubleshooting guide. Extra Info: host=$host; status=$status;"
```

Comment:
can we put in config file instead of const

Reply:
As I mentioned in my code comment above, this is to be removed. Yes, I think it will go to some config file which would come through other alert generated tasks. I will merge then and remove this current implementation.